error: null
name: bioimageio.spec static validation
nested_errors: {}
source_name: '{name: CREMI: MICCAI Challenge on Circuit Reconstruction from Electron
  Microscopy Images, ...}'
traceback: null
warnings: {type: Unrecognized type dataset. Validating as rdf.}
